integrations/langchain/README.md

```diff
@@ -103,38 +112,128 @@ if __name__ == "__main__":
 
 no need to run `skyvern init` command in your terminal to set up skyvern before using this integration.
 
+```python
+import asyncio
+from skyvern_langchain.client import RunTask
+
+run_task = RunTask(
+    credential="<your_organization_api_key>",
+)
+# or you can load the credential from SKYVERN_CREDENTIAL in .env
+# run_task = RunTask()
+
+async def main():
+    print(await run_task.ainvoke("Navigate to the Hacker News homepage and get the top 3 posts."))
+
+
+if __name__ == "__main__":
+    asyncio.run(main())
+```
+
+### Run a task(async) with skyvern client (calling skyvern OpenAPI in the tool)
+> async task will return immediately and the task will be running in the background.
+
+no need to run `skyvern init` command in your terminal to set up skyvern before using this integration.
+
+the task is actually running in the skyvern cloud service, so you don't need to keep your script running until the task is finished.
+
+```python
+import asyncio
+from skyvern_langchain.client import DispatchTask
+
+dispatch_task = DispatchTask(
+    credential="<your_organization_api_key>",
+)
+# or you can load the credential from SKYVERN_CREDENTIAL in .env
+# dispatch_task = DispatchTask()
+
+async def main():
+    print(await dispatch_task.ainvoke("Navigate to the Hacker News homepage and get the top 3 posts."))
+
+
+if __name__ == "__main__":
+    asyncio.run(main())
+```
+
+
+### Get a task with skyvern client (calling skyvern OpenAPI in the tool)
+> async task will return immediately and the task will be running in the background.
+
+no need to run `skyvern init` command in your terminal to set up skyvern before using this integration.
+
+the task is actually running in the skyvern cloud service, so you don't need to keep your script running until the task is finished.
+
+```python
+import asyncio
+from skyvern_langchain.client import GetTask
+
+get_task = GetTask(
+    credential="<your_organization_api_key>",
+)
+# or you can load the credential from SKYVERN_CREDENTIAL in .env
+# get_task = GetTask()
+
+async def main():
+    print(await get_task.ainvoke("<task_id>"))
+
+
+if __name__ == "__main__":
+    asyncio.run(main())
+```
+
+## Agent Usage
+
+Langchain is more powerful when used with [Langchain Agents](https://python.langchain.com/v0.1/docs/modules/agents/). Here is an example of how to use skyvern langchain tool with agent.
+
+
+### Run a task(async) and wait until the task is finished by skyvern agent (calling skyvern agent function directly in the tool)
```

Comment:
1. Title is too long. this part is more like description.

2. thoughts on writing better description: we should the scenario of what this agent is doing. Like `This code builds an agent that executes a specified task, waits for its completion, and then returns the results. For example, the agent is tasked with navigating to the Hacker News homepage and retrieving the top three posts.`